packages/atchops/include/atchops/hex.h

```diff
@@ -0,0 +1,38 @@
+#ifndef UTILS_H
+#define UTILS_H
```

Comment:
Macro guards for pragma once should follow the naming scheme of `<PACKAGENAME>_<FILENAME>` e.g. `ATCHOPS_HEX_H`